pyproject.toml

```diff
@@ -60,8 +63,4 @@ Zope = [
   'Products.CMFDynamicViewFTI', 'zope.deprecation',
 ]
 python-dateutil = ['dateutil']
-beautifulsoup4 = ['bs4', ]
-ignore-packages = ['zope.app.component']
```

Comment:
When running `tox -e dependencies` it does not complain about `zope.app.component`? 🤔 

Maybe you removed the reference within this PR?

Reply:
I think it complained, but I saw `zope.app.component` was only used as backwards compatibility import when `zope.component.hooks` was not found, so I removed it from that code.